fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/serialization/LoanApplicationValidator.java

```diff
@@ -2127,4 +2148,22 @@ public static void validateOrThrow(String resource, Consumer<DataValidatorBuilde
                     dataValidationErrors);
         }
     }
+
+    private Long setOfficeId(Client client, Group group) {
```

Comment:
It still not sets anything. Please rename it! Example: `fetchOfficeId` or something would be more descriptive...

Reply:
Very much agreed. I wouldn't even say "fetch" because it suggests fetching from an external source (like the DB).

I'd rather call it something like "resolveOfficeId". Also, probably this wouldn't be the place to put such a method because in my opinion this is a very generic way to get the office ID for a specific client.

Reply:
Done! method name and logic updated